src/components/store/CollectionSlider/index.tsx

```diff
@@ -0,0 +1,34 @@
+import ItemCard from '@/components/store/ItemCard';
+import { config } from '@/lib';
+import { PublicMerchItem } from '@/lib/types/apiResponses';
+import NoImage from '@/public/assets/graphics/cat404.png';
```

Comment:
I'm using the 404 cat image temporarily. We'll probably want to replace it with something else, though I don't imagine this image showing up in practice.